pytorch_lightning/loggers/base.py

```diff
@@ -9,6 +9,7 @@
 import torch
 
 from pytorch_lightning.utilities import rank_zero_only
+from omegaconf import DictConfig
```

Comment:
`omegaconf` is not mandatory dependency
```suggestion
try:
    from omegaconf import Container
except ImportError:
    Container = None
```